hazelcast/src/main/java/com/hazelcast/map/impl/operation/MapOperationProvider.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.map.impl.operation;
+
+import com.hazelcast.map.EntryProcessor;
+import com.hazelcast.map.impl.MapEntries;
+import com.hazelcast.nio.serialization.Data;
+import com.hazelcast.query.Predicate;
+import com.hazelcast.spi.OperationFactory;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Provides {@link com.hazelcast.config.InMemoryFormat InMemoryFormat} specific
+ * operations for {@link com.hazelcast.core.IMap IMap}.
+ */
+//CHECKSTYLE:OFF
```

Comment:
Don't do this. If you have too many methods, just suppress the number of methods. But suppressing anything on the whole class is no good because you also suppress warnings that do make sense.